scripts/install_third_party.py

```diff
@@ -338,10 +339,74 @@ def download_manifest_files(filepath):
                     dependency_tar_root_name, dependency_target_root_name)
 
 
+def install_redis_cli():
+    """This installs the redis-cli to the local oppia third_party directory so
+    that development servers and backend tests can make use of a local redis
+    cache. Redis-cli installed here (redis-cli-6.0.6) is different from the
+    redis package installed in manifest.json (redis-3.5.3). The redis-3.5.3
+    package detailed in manifest.json is the Python library that allows users to
+    communicate with any Redis cache using Python. The redis-cli-6.0.6 package
+    installed in this function contains C++ scripts for the redis-cli and
```

Comment:
This is a much better explanation; thanks.

One question: is the intention to deploy redis-cli to prod GAE? Similarly, does redis-3.5.3 need to be deployed? I ask because: if you're deploying it, then keep it in third_party, but if not, it should go in oppia_tools. Only stuff in the former directory is deployed, and to my knowledge prod GAE can't run C++ scripts.

Reply:
Done